### ISSUE TITLE
Add config option for disabling sandbox in lua (1.5)

### ISSUE THREAD

**Issue by Mons** [tarantool/tarantool#705]
To make normal, reloadable application, I need an option to keep 'require' etc alive.
Otherwise my common  beginning of init.lua looks like
```
if not bkp then rawset(_G,'bkp',{_first = true}) end
if bkp._first then
    bkp.package = package
    bkp.require = require
    bkp._first   = false
    bkp.first    = true
else
    bkp.first    = false
    package = bkp.package
    require = bkp.require
end
```

It could not be shipped as a module and should be copypasted into every instance.

**Comment by kostja** [comment]
Fixed in 1.5.4-58-g4f21d0d

**Comment by kostja** [comment]
Set box.sandbox to nil to disable sandbox.
